common/source/docs/common-polygon_fence.rst

```diff
@@ -1,64 +1,138 @@
 
--  Connect your Autopilot to the Mission Planner
--  Go to the **CONFIG\| GeoFence** screen
--  Click "Enable" and set Type to "All" or another option that includes "Polygon". Circle refers to the HOME centered, :ref:`cylindrical fence <common-ac2_simple_geofence>`. Altitude only, limits maximum altitude (everywhere).
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit you want (in meters). This is unavailable in Rover.
+-  Set :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 to enable FENCEs.
+-  Set :ref:`FENCE_MARGIN<FENCE_MARGIN>` = to the distance from the fence horizontal boundary the vehicle must maintain in order to prevent a breach.
+-  Set :ref:`FENCE_RADIUS<FENCE_RADIUS>` to the maximum distance from HOME you want (in meters). This should normally be at least 50m. This value must be larger than :ref:`FENCE_MARGIN<FENCE_MARGIN>` and greater than 30m.
+-  :ref:`FENCE_TOTAL<FENCE_TOTAL>` should not be changed manually and indicates the number of inclusion/exclusion fence points stored. (See :ref:`common-polygon_fence`)
+-  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = is a bitmap set to enable the various fence types: MIN or MAX altitude, simple CIRCLE tin can around HOME, or POLYGON fences. The POLYGON fences must also have been loaded via a fence list from a ground control station in order to be active.
 
+.. note:: Polygon fence type includes the circular fences specified in the Inclusion/Exclusion fence list. The simple home centered CIRCLE fence is a separate fence. Rover ignores altitudes, if set.
 
-   .. image:: ../../../images/polygon_fence_enable.png
-       :target: ../_images/polygon_fence_enable.png
-       :width: 500px
+Defaults for :ref:`FENCE_TYPE<FENCE_TYPE>` are:
 
--  Leave the Action as "RTL or Land". Although, you can select other failsafe actions.
--  Go to the **PLAN** screen, and select ``FENCE`` from the dropdown box in the upper right part of the map display.
--  For a circular fence, simply click on the map and a circular exclusion fence is added to the list. You can change its radius with the first parameter of the item. You can also change it from an exclusion fence to a circular inclusion fence, via the item's dropdown box.
+- Rover: CIRCLE and POLYGON
+- Copter: ALT MAX, CIRCLE, and POLYGON
+- Plane: POLYGON
 
-   .. image:: ../../../images/exclusion-zone.png
-       :target: ../_images/exclusion-zone.png
-       :width: 500px
+If Plane, you can also:
 
+-  Set the :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>` as a breach boundary.
+-  Set :ref:`FENCE_AUTOENABLE<FENCE_AUTOENABLE>` = to allow automatic temporary (until reboot)forcing of :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 under certain vehicle conditions, such as upon arming or takeoff. A value of 0 disables this feature.
+-  :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` allows returning to the nearest RALLY point (See:), if loaded, instead of HOME.
```

Comment:
What am I "seeing"? Is it the video below?

```suggestion
-  :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` allows returning to the nearest RALLY point (See: video below), if loaded, instead of HOME.
```